kgforge/core/commons/files.py

```diff
@@ -32,7 +36,7 @@ def load_file_as_byte(source: str):
         data = filepath.read_bytes()
     else:
         try:
-            response = requests.get(source)
+            response = requests.get(source, timeout=3600)
```

Comment:
I guess this is the only call that needs that much time?

Reply:
It's a mistake, fixed it!